examples/plot_fmri_data_example.py

```diff
@@ -151,19 +154,26 @@ def preprocess_haxby(subject=2, memory=None):
 
 SVR_permutation_test_inference = False
 if SVR_permutation_test_inference:
-    # We computed the regularization parameter by CV (C = 0.1)
-    pval_corr_svr_perm_test, one_minus_pval_corr_svr_perm_test = permutation_test_cv(
-        X, y, n_permutations=50, C=0.1
+    # We computed the regularization parameter from a cross valisation (C = 0.001)
+    estimator = LinearSVR(C=0.001)
+    weight_svr, weight_svr_distribution = permutation_test(
+        X, y, estimator, n_permutations=50
+    )
+    pval_corr_svr_perm_test, one_minus_pval_corr_svr_perm_test = permutation_test_pval(
+        weight_svr, weight_svr_distribution
     )
 
 # Another method is to compute the p-values by permutation test from the
 # Ridge decoder. The solution provided by this method should be very close to
 # the previous one and the computation time is much shorter: around 20 seconds.
-
-estimator = Ridge()
-pval_corr_ridge_perm_test, one_minus_pval_corr_ridge_perm_test = permutation_test(
+# We computed the parameter from a cross valisation (alpha = 0.0215)
```

Comment:
oops. Where do we get these value from ?

Reply:
This value comes from a cross-validation of the model. From the previous discussion, we say that we don't include cross-validation.

Reply:
OK, but that sounds like a mistake (very likely on my side, sorry for that). I think we should make it explicit how we set hyperparameters. If this does not increase the cost too much, we should really do it. Magic numbers are something we want to avoid in any circumstance.